xml/ha_storage_basics.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<!-- Converted by suse-upgrade version 1.1 -->
+<chapter xmlns="http://docbook.org/ns/docbook" 
+xmlns:xi="http://www.w3.org/2001/XInclude" 
+xmlns:xlink="http://www.w3.org/1999/xlink" 
+version="5.0" xml:id="cha.ha.storage.dlm">
+ <title>Distributed Lock Manager (DLM)</title>
+ <info>
+  <abstract>
+   <para>
+    The Distributed Lock Manager (DLM) coordinates disk access for clustered
+    file systems. It administers file locking to increase performance and
+    availability. The DLM in the kernel is the base component used by &ocfs;,
+    GFS2, cluster MD, and cluster LVM (lvmlockd) to provide active-active
+    storage at each respective layer. The Distributed Lock Manager (DLM)
+    coordinates disk access for clustered file systems. It administers file
+    locking to increase performance and availability. The DLM in the kernel is
+    the base component used by &ocfs;, GFS2, cluster MD, and cluster LVM
+    (lvmlockd) to provide active-active storage at each respective layer.
+   </para>
+  </abstract>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <sect1 xml:id="sec.ha.storage.dlm.protocol">
+  <title>Protocols for DLM Communication</title>
+
+  <para>
+   DLM can either use the TCP or SCTP protocol for communication. If network
+   device bonding cannot be used for any reason, the second best choice is to
+   define a redundant communication channel (a second ring) in &corosync;. If
+   you have configured redundant communication rings in &corosync.conf;, DLM
+   will automatically SCTP. However, if you want DLM to use TCP for
+   communication, disable <guimenu>rrp_mode</guimenu> by setting it to
+   <literal>none</literal>. For details on configuring a redundant
+   communication channel in &corosync; and setting <guimenu>rrp_mode</guimenu>,
+   see <xref linkend="pro.ha.installation.setup.channel2"/>.
+  </para>
+ </sect1>
+ <sect1 xml:id="sec.ha.storage.generic.dlm.config">
+  <title>Configuring DLM Cluster Resources</title>
+
+  <para>
+   DLM uses the cluster membership services from Pacemaker which run in user
+   space. Therefore, DLM needs to be configured as clone resource that is
+   present on each node in the cluster.
+  </para>
+
+  <note>
+   <title>DLM Resource for Several Solutions</title>
+   <para>
+    As &ocfs;, GFS2, cluster MD, and cluster LVM (lvmlockd) all use DLM, it is
+    enough to configure one resource for DLM. As the DLM resource runs on all
+    nodes in the cluster it is configured as a clone resource.
+   </para>
+   <para>
+    If you have a setup that includes both &ocfs; and LVM2, configuring
```

Comment:
1. Strictly speaking, only Cluster LVM2 matters to DLM. The reason is the native LVM2 has nothing to do with DLM ;)

2. The standard term for Cluster LVM2 and Cluster MD is better to be consistent for the whole Admin Guide. I saw some captal Cluster which I prefer, while some lower case "cluster". We intentionally use Cluster LVM2 to intentially differentiate with cLVM2. That says, Cluster LVM2 can refers to both cLVM2 and lvmlockd.